dao/src/main/java/com/iluwatar/dao/App.java

```diff
@@ -11,43 +13,38 @@
  */
 public class App {
 
+	private static Logger LOGGER = Logger.getLogger(App.class);
+
 	/**
-	 * Program entry point
-	 * @param args command line args
+	 * Program entry point.
+	 * 
+	 * @param args command line args.
 	 */
-	public static void main(String[] args) {
-
-		CustomerDaoImpl customerDao = new CustomerDaoImpl(generateSampleCustomers());
-
-		System.out.println("customerDao.getAllCustomers(): " + customerDao.getAllCustomers());
-		System.out.println("customerDao.getCusterById(2): " + customerDao.getCusterById(2));
-
-		Customer customer = new Customer(4, "Dan", "Danson");
+	public static void main(final String[] args) {
+		final CustomerDaoImpl customerDao = new CustomerDaoImpl(generateSampleCustomers());
+		LOGGER.info("customerDao.getAllCustomers(): " + customerDao.getAllCustomers());
+		LOGGER.info("customerDao.getCusterById(2): " + customerDao.getCustomerById(2));
+		final Customer customer = new Customer(4, "Dan", "Danson");
 		customerDao.addCustomer(customer);
```

Comment:
Using final everywhere just adds noise IMHO. You might want to use final when it matters.